wrappers/angular-handsontable/README.md

```diff
@@ -0,0 +1,192 @@
+<div align="center">
+  
+![Handsontable for Angular](https://raw.githubusercontent.com/handsontable/static-files/master/Images/Logo/Handsontable/handsontable-angular.png)
+
+This is the official wrapper of [**Handsontable**](//github.com/handsontable/handsontable) data grid for Angular.<br>
+It provides data binding, data validation, filtering, sorting and more.<br>
+
+[![npm](https://img.shields.io/npm/dt/@handsontable/angular.svg)](//npmjs.com/package/@handsontable/angular)
+[![npm](https://img.shields.io/npm/dm/@handsontable/angular.svg)](//npmjs.com/package/@handsontable/angular)
+[![Build status](https://travis-ci.org/handsontable/angular-handsontable.png?branch=master)](//travis-ci.org/handsontable/angular-handsontable)
```

Comment:
After the changes, the TravicCI badge can be replaced with the CodeShip badge, if they have any 😅